ITEM.tsv

```diff
@@ -20,13 +20,13 @@ ITEM_20150317_000019	Due to its ease in manufacturing, this sword is frequently
 ITEM_20150317_000020	Cinquedea
 ITEM_20150317_000021	Recommended for those who desire a heavier and stronger sword.
 ITEM_20150317_000022	Sabre
-ITEM_20150317_000023	Primary weapon of the cavalry optimized for slashing. The Hilt also protects your hands.
+ITEM_20150317_000023	Primary weapon of the cavalry optimized for slashing. It features a guard on the hilt to protect the wielder's hands.
 ITEM_20150317_000024	Shamshir
 ITEM_20150317_000025	The large curved crescent edge allows for a powerful slash.
 ITEM_20150317_000026	Scimitar
-ITEM_20150317_000027	Its heavy weight is compensated by a very powerful slash.
+ITEM_20150317_000027	Its heavy weight is compensated by its very powerful slash.
 ITEM_20150317_000028	Kris
-ITEM_20150317_000029	Both practical and decorative, its curved edge amplifies the damage it inflicts.
+ITEM_20150317_000029	A dagger that is both practical and decorative. Its wavy blade amplifies the damage it inflicts.
```

Comment:
Once again overcrowding text. "Both practical and decorative, the dagger's curved edge amplifies the damage it inflicts."